plugins/inputs/jti_openconfig_telemetry/sample.conf

```
@@ -33,6 +33,11 @@
    "/interfaces",
   ]
 
+  ## Timestamp Source
+  ## Set to 'current' for time of collection, and 'data' for using the time
+  ## provided by the _timestamp field.
+  # timestamp_source = "current"
```

Maybe
```suggestion
  ## Set to 'collection' for time of collection, and 'data' for using the time
  ## provided by the _timestamp field.
  # timestamp_source = "collection"
```
would be more clear?

---

I like it - done